Add support for image popovers (#854)
jackyzha0/quartz: docs/features/popover previews.md

When [[creating components|creating your own components]], you can include this `popover-hint` class to also include it in the popover.

> Like in Obsidian, [[quartz layout.png|images referenced using wikilinks]] can also be viewed as popups.

Reviewer: ```suggestion
Similar to Obsidian, [[quartz layout.png|images referenced using wikilinks]] can also be viewed as popups.
```
